src/testFixtures/java/org/opensearch/knn/TestUtils.java

```diff
@@ -68,7 +68,15 @@ public class TestUtils {
     public static final String OPENDISTRO_SECURITY = ".opendistro_security";
     public static final String BWCSUITE_CLUSTER = "tests.rest.bwcsuite_cluster";
     public static final String BWCSUITE_ROUND = "tests.rest.bwcsuite_round";
+    public static final String BWC_VERSION = "tests.plugin_bwc_version";
+    public static final String MIXED_CLUSTER = "mixed_cluster";
+    public static final String NODES_BWC_CLUSTER = "3";
+    public static final String OLD_CLUSTER = "old_cluster";
+    public static final String RESTART_UPGRADE_OLD_CLUSTER = "tests.is_old_cluster";
+    public static final String ROLLING_UPGRADE_FIRST_ROUND = "tests.rest.first_round";
     public static final String TEST_CLUSTER_NAME = "tests.clustername";
+    public static final String TIMEOUT_90s = "90s";
```

Comment:
nit: can we keep either 90_SECONDS as a general time unit or timeout specific name (extended timeout or similar)?